financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/main/java/org/wso2/financial/services/accelerator/common/config/FinancialServicesConfigParser.java

```diff
@@ -160,25 +161,26 @@ private void buildFSExecutors() {
                             FinancialServicesConstants.GATEWAY_EXECUTOR_CONFIG_TAG));
 
             if (financialServicesGatewayExecutors != null) {
-                //obtaining each consent type element under FinancialServicesGatewayExecutors tag
+                // obtaining each consent type element under FinancialServicesGatewayExecutors
+                // tag
                 Iterator consentTypeElement = financialServicesGatewayExecutors.getChildElements();
                 while (consentTypeElement.hasNext()) {
                     OMElement consentType = (OMElement) consentTypeElement.next();
                     String consentTypeName = consentType.getLocalName();
                     Map<Integer, String> executors = new HashMap<>();
-                    //obtaining each Executor element under each consent type
+                    // obtaining each Executor element under each consent type
                     Iterator<OMElement> obExecutor = consentType.getChildrenWithName(
```

Comment:
should we rename this to something other than ob?

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/110/commits/fe1e2b6aa311de998ac4f0028ad72f9247c4dd0c